Src/Support/Google.Apis.Auth/OAuth2/ImpersonatedCredential.cs

```diff
@@ -120,9 +155,9 @@ public override async Task<bool> RequestAccessTokenAsync(CancellationToken taskC
         {
             var request = new ImpersonationAccessTokenRequest
             {
-                DelegateAccounts = Options.DelegateAccounts,
-                Scopes = Options.Scopes,
-                Lifetime = ((int)Options.Lifetime.TotalSeconds).ToString() + "s"
+                DelegateAccounts = DelegateAccounts,
+                Scopes = Scopes,
+                Lifetime = $"{(int)Lifetime.TotalSeconds}s"
```

Comment:
Do we validate anywhere that this is non-negative? It's probably worth doing so - partly so that we don't need to worry about different culture settings affecting what this does. (We could also use `Invariant ($...)` with an appropriate static import of FormattableString.)